spring-native/src/main/java/org/springframework/aot/test/boot/AotSpringBootConfigContextLoader.java

```diff
@@ -49,6 +55,23 @@
  */
 public class AotSpringBootConfigContextLoader extends SpringBootContextLoader {
 
+	// "SpringBootTestArgs#get(Set)" static method
+	private static final Method SPRING_BOOT_TEST_ARGS__GET;
```

Comment:
Rather than using reflection, I'd rather add an accessor class in `spring-native` the same way we do elsewhere (a class in the same package that exposes the package-private behavior).